content/faqs/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: FAQs
+weight: 10
+---
+
+Teams often ask us questions, so we've put the most common ones below, so others can learn from them.
+
+{{% expand "Do these FAQs work?" %}}
```

Comment:
Is it worth coming up with a couple of 'real' FAQs instead of just dummy data?

Reply:
I agree, right now this page looks like a test page (even though we intend to add useful content at a later date)

Reply:
Well, that's technically what it is. But i'll try and come up with some generic things to fill it out a bit